index.d.ts

```diff
@@ -9,9 +37,29 @@ export interface Options {
 	/**
 	Number of milliseconds to wait before automatically rejecting with a `TimeoutError`.
 
+	You can customize the timeout `Error` by specifying `TimeoutOptions`.
+
 	@default Infinity
+
+	@example
+	```js
+	import pWaitFor from 'p-wait-for';
+	import {pathExists} from 'path-exists';
+
+	await pWaitFor(() => pathExists('unicorn.png'), {
+		timeout: {
+			milliseconds: 100,
+			message: new MyError('Time’s up!'),
+			customTimers: {
+				setTimeout: requestAnimationFrame
```

Comment:
@sindresorhus It this was copied, it's a bad example since rFA does not accept a second (milliseconds) parameter 🥲

Reply:
@jopemachine This should be fixed here and at the source.

Reply:
> This should be fixed here and at the source.

I think I don't know a good example of `customTimers` other than [p-timeout's one](https://github.com/jopemachine/p-timeout/blob/main/readme.md#customtimers).

Could we reuse this example here?